contracts/token/ERC20/extensions/ERC20Capped.sol

```diff
@@ -28,10 +28,17 @@ abstract contract ERC20Capped is ERC20 {
     }
 
     /**
-     * @dev See {ERC20-_mint}.
+     * @dev See {ERC20-_transfer}.
      */
-    function _mint(address account, uint256 amount) internal virtual override {
-        require(ERC20.totalSupply() + amount <= cap(), "ERC20Capped: cap exceeded");
-        super._mint(account, amount);
+    function _update(
+        address from,
+        address to,
+        uint256 amount
+    ) internal virtual override {
+        if (from == address(0)) {
+            require(ERC20.totalSupply() + amount <= cap(), "ERC20Capped: cap exceeded");
```

Comment:
What is the rational for using `ERC20.totalSupply()` instead of `totalSupply()` ?